web-client/example/README.md

```diff
@@ -24,7 +24,7 @@ to build the WASM files. Then go into `example/node` and run
 yarn
 ```
 
-to install the dependencies. Other package managers will also work, but the `preinstall` script uses `yarn`.
+to install the dependencies. Other package managers will also work.
```

Comment:
Yarn isn't mandatory all together anymore?

Reply:
Not here, as I removed the postinstall script. So packages can be installed with any manager for the example.